tests/unit/store/Account.spec.js

```diff
@@ -0,0 +1,280 @@
+const {
+  randomHex,
+  randomAccountId,
+  randomAssetId,
+  randomNodeIp,
+  randomPrivateKey,
+  randomObject,
+  randomAmount
+} = helper
+const expect = chai.expect
+
+chai.use(require('chai-things'))
+
+describe('Account store', () => {
+  /*
+   * Mock irohaUtil so that unit tests can work without irohad
+   */
+  const MOCK_ASSETS = require('../fixtures/assets.json')
+  const MOCK_ASSET_TRANSACTIONS = require('../fixtures/transactions.json')
+  const MOCK_TRANSACTIONS = MOCK_ASSET_TRANSACTIONS['bitcoin#test']
+  const MOCK_NODE_IP = 'MOCK_NODE_IP'
+  const MOCK_ACCOUNT_RESPONSE = { accountId: randomAccountId() }
+  const irohaUtil = require('../../../src/util/iroha-util')
+  const irohaUtilMock = Object.assign(irohaUtil, {
+    getStoredNodeIp: () => MOCK_NODE_IP,
+    login: (username, privateKey, nodeIp) => Promise.resolve(MOCK_ACCOUNT_RESPONSE),
+    logout: () => Promise.resolve(),
+    getAccountAssetTransactions: () => Promise.resolve(MOCK_TRANSACTIONS),
+
+    // TODO: fix it after irohaUtil.getAccountAssets is updated
+    getAccountAssets: (accountId, assetId) => {
+      return Promise.resolve(MOCK_ASSETS.find(a => a.accountAsset.assetId === assetId))
+    },
+
+    transferAsset: () => Promise.resolve()
+  })
+
+  let types, mutations, actions, getters
+
+  beforeEach(() => {
+    ({ types, mutations, actions, getters } = AccountInjector({
+      'util/iroha-util': irohaUtilMock,
+      'util/iroha-amount': require('../../../src/util/iroha-amount')
+    }).default)
+  })
+
+  describe('Mutations', () => {
+    function testErrorHandling (type) {
+      const codes = ['UNAVAILABLE', 'CANCELLED']
+
+      codes.forEach(codeName => {
+        it(`${type} should treat grpc ${codeName} as a connection error`, () => {
+          const grpc = require('grpc')
+          const state = {}
+          const error = { code: grpc.status[codeName] }
+
+          expect(state.connectionError).to.not.exist
+
+          mutations[types[type]](state, error)
+
+          expect(state.connectionError).to.equal(error)
+        })
+      })
+
+      it(`${type} should not treat other errors as a connection error`, () => {
+        const state = {}
+        const error = new Error()
+
+        expect(state.connectionError).to.not.exist
+
+        mutations[types[type]](state, error)
+
+        expect(state.connectionError).to.not.exist
+      })
+    }
+
+    it('RESET should reset the state', () => {
+      const state = {
+        accountId: randomAccountId(),
+        nodeIp: randomNodeIp(),
+        accountInfo: randomObject(),
+        rawAssetTransactions: randomObject(),
+        assets: randomObject(),
+        connectionError: new Error()
+      }
+      const expectedState = {
+        accountId: '',
+        nodeIp: MOCK_NODE_IP,
+        accountInfo: {},
+        rawAssetTransactions: {},
+        assets: [],
+        connectionError: null
+      }
+
+      mutations[types.RESET](state)
+
+      expect(state).to.deep.equal(expectedState)
```

Comment:
`.to.deep` => `.to.be.deep` in several places (not a mistake of course, just small comment)